test.js

```diff
@@ -160,15 +160,25 @@ test('new VFile(options?)', async function (t) {
     )
 
     assert.equal(
-      new VFile(Buffer.from('bar')).toString(),
-      'bar',
-      'buffer: should return the internal value'
+      new VFile(Buffer.from([0xef, 0xbb, 0xbf, 0x61, 0x62, 0x63])).toString(),
```

Comment:
does this mean buffer from string is no longer supported, and only hex can be used?